web/client/components/misc/GridCard.jsx

```diff
@@ -24,6 +24,16 @@ const GridCard = React.createClass({
             header: ""
         };
     },
+    // click on the card triggers the first available action
+    onClick() {
+        // Users can select Title and Description without triggering the click
```

Comment:
No, all the card area is clickable and will open the map.
Only if the user highlights something the map will not open.